tendrl/notifier/notification/handlers/mail_handler.py

```diff
@@ -173,7 +176,20 @@ def get_mail_client(self):
     def dispatch_notification(self, alert):
         server = None
         try:
-            self.set_destinations()
+            user_ids = self.set_destinations()
```

Comment:
`get_destinations` ?

Reply:
yes it will return a list of user_ids who are all enabled email notification